Final_project_2/frontend/css/style.css

```diff
@@ -463,20 +463,15 @@ label  {
   margin-right: 30px;
 } 
 
-
-
 #contactform {
-  padding: 80px;
-  
+    display: flex;
+    padding-left: 80px;
+    padding-right: 80px;
 }
 
-.fanmail {
-  flex-direction: column;
-  width: 100%;
-  text-align: center;
-  margin-top: 80px;
-  margin-bottom: 180px;
-  font-size: 14px;
+.address-block{
```

Comment:
This is a new class to style the address block to be all in the centre of the page. The two properties work together to achieve that.